add all team with triage permissions

vito opened the pull request:
fixes #35

- import github labels
- remove team.Legacy
- add contributors team
- harmonize: begin backfilling tests
- harmonize: test synchronizing role position too
- harmonize: switch to new delta package

vito committed:
import github labels

doesn't really matter since they're created idempotently, but don't want
it to seem like we forgot about it

Signed-off-by: Alex Suraci <asuraci@vmware.com>

vito committed:
remove team.Legacy

the legacy teams are no more!

Signed-off-by: Alex Suraci <asuraci@vmware.com>

vito committed:
add contributors team

* teams can set all_contributors: true to include all contributors
* teams can set repo_permission to override the default 'maintain'
  permission granted for repos

Signed-off-by: Alex Suraci <asuraci@vmware.com>

vito committed:
harmonize: begin backfilling tests

* rename 'contributors' back to 'all' - forgot i can just set the role
  name! i think @concourse/all is a nicer team name.
* add 'sticky: true' so we're not special-casing the contributors team
  by name.

TODO: actually implement Delta.Apply everywhere, and integrate it into
cmd/harmonize.

Signed-off-by: Alex Suraci <asuraci@vmware.com>

vito committed:
harmonize: test synchronizing role position too

Signed-off-by: Alex Suraci <asuraci@vmware.com>

vito committed:
harmonize: switch to new delta package

Signed-off-by: Alex Suraci <asuraci@vmware.com>

clarafu reviewed (approved):
I think iz coolio poolio

vito committed:
harmonizer: add logging to Delta

would be nice to use a context.Context here instead of *zap.Logger, but
don't want to introduce a whole zapctx package just for this

Signed-off-by: Alex Suraci <asuraci@vmware.com>
